pkg/reconciler/instances/ory/action_test.go

```diff
@@ -50,16 +51,112 @@ const (
 const (
 	profileName   = "profile"
 	componentName = "test-ory"
+	inMemoryUrl   = "sqlite://file::memory:?cache=shared&busy_timeout=5000&_fk=true"
 )
 
 var chartDir = filepath.Join("test", "resources")
 
+func Test_PostReconcile_Run(t *testing.T) {
+	t.Parallel()
+	t.Run("should call hydra sync if we are inMemory mode", func(t *testing.T) {
```

Comment:
General comment - please use `when` instead of `if` in test cases definitions - it applies to the rest of the tests too